src/event-manager/event-channel-client.ts

```diff
@@ -1,5 +1,5 @@
 import Ain from '../ain';
-import * as WebSocket from 'isomorphic-ws';
```

Comment:
could you leave the original code with commenting out for future references?

Reply:
done!

Reply:
ah.. I mean the code in event-channel-client.ts.

Reply:
@platfowner okay, done!